src/input.rs

```diff
@@ -1155,6 +1156,7 @@ fn extract_by_ilp(
     let order_var_int = false;
     let class_constraint = false;
     let no_order = true;
+    let fusion_costs: bool = std::env::var("FUSION_COSTS").unwrap_or(String::from("false")).parse().unwrap();
```

Comment:
not sure if the default should be true or false. if it's not worse then maybe we should default to true (with only using false for benchmarking)

Reply:
or alternatively have `NO_FUSION_COSTS` or something

Reply:
🤠